src/main/java/com/znsio/teswiz/runner/Driver.java

```diff
@@ -37,7 +37,7 @@
 import java.util.Collections;
 import java.util.List;
 import java.util.Set;
-
+import java.util.Collection;
```

Comment:
you can use single import here like this java.util.*;
and remove other imports and cross check for errors as well afterwards

Reply:
Looks like I have a different understanding on this one, one more teammate review will be required here

Reply:
Keep individual imports